accounts-db/src/verify_accounts_hash_in_background.rs

```diff
@@ -81,14 +73,11 @@ impl VerifyAccountsHashInBackground {
             // already completed
             return true;
         }
-        if self.complete.wait_timeout(Duration::default())
```

Comment:
so this was always a wait(0), which checks and returns? Yep, we can get rid ofit.

Reply:
yeah.